iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuizSubmodules/StepQuizHints/StepQuizHintsView.swift

```diff
@@ -0,0 +1,118 @@
+import shared
+import SVProgressHUD
+import SwiftUI
+
+extension StepQuizHintsView {
+    struct Appearance {
+        let skeletonInitialSize = CGSize(width: 146, height: 34)
```

Comment:
Very nice! Now we won't have problem with changing content size when load and show hint button at the first time 👍